data/moves.ts

```diff
@@ -1118,8 +1118,8 @@ export const Moves: {[moveid: string]: MoveData} = {
 		pp: 40,
 		priority: 0,
 		flags: {},
-		onHit(target) {
-			if (!this.canSwitch(target.side)) {
+		onTryHit(target) {
```

Comment:
Did this change as well? Baton Pass failures weren't this early in previous generations.

Reply:
Ah sorry, the onHit failure caused a strange issue where the commanded dondozo could still switch out even though it triggered a failure. I can refactor the commander failure to be somewhere else if needed.

Reply:
Apparently we're using the same thing for Healing Wish and Lunar Dance to get them to work, so I'll just merge now since it's only a slight mechanics error and fixes a more important one.